internal/provider/tests/unit/hypercore_iso_resource_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package unit
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-hypercore/internal/provider"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHypercoreISOResource_Schema(t *testing.T) {
```

Comment:
This function tests if schema is loaded, and if description strings are "correct".
Schema loading is tested also in mock acceptance tests.
I do not see much sense in testing if description strings are "correct".

We can leave this as an example, for some future unit tests.